hexrdgui/resources/ui/main_window.ui

```diff
@@ -58,8 +58,15 @@
      <property name="toolTipsVisible">
       <bool>true</bool>
      </property>
+     <widget class="QMenu" name="menu_state">
+      <property name="title">
+       <string>State</string>
+      </property>
+      <addaction name="action_open_state"/>
+      <addaction name="separator"/>
+     </widget>
```

Comment:
I personally think that the list of state files should live under a new menu item under "File" called "Open Recent", similar to here:

![image](https://github.com/HEXRD/hexrdgui/assets/9558430/4c32079d-67a8-48a9-8b33-edfd42f0d101)

And that we should move "action_open_state" back to how it was before.

This is also similar to what we did in Tomviz:

![image](https://github.com/HEXRD/hexrdgui/assets/9558430/46877abf-56c2-4c81-9a14-06440276fca9)

It's okay that it's just state files and not any other type of file.

What do you think, @saransh13 @akrygier

Reply:
I had originally started with things this way but I changed it since we are only remembering state files at the moment. But I do like this better overall so if it's not too confusing and it's good with @saransh13 and @akrygier then it's good with me!